samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -38,11 +38,6 @@ Provisioning is performed using the `nRF Mesh mobile app`_.
 This mobile application is also used to configure key bindings, and publication and subscription settings of the Bluetooth mesh model instances in the sample.
 After provisioning and configuring the mesh models supported by the sample in the `nRF Mesh mobile app`_, you can control the dimmable LED on the development kit from the app.
 
-.. note::
```

Comment:
Just to clarify: @mia-ko added this entry approximately two months ago, and now we are removing it without any changes to the light application code. Does that mean that the light sample always has conducted the correct behaviour?